src/Tizen.Security.SecureRepository/Tizen.Security.SecureRepository/DataManager.cs

```diff
@@ -55,15 +55,53 @@ static public byte[] Get(string alias, string password)
 
             try
             {
-                Interop.CheckNThrowException(
-                    Interop.CkmcManager.GetData(alias, password, out ptr),
+                CheckNThrowException(
+                    CkmcManager.GetData(alias, password, out ptr),
                     "Failed to get certificate. alias=" + alias);
                 return new SafeRawBufferHandle(ptr).Data;
             }
             finally
             {
                 if (ptr != IntPtr.Zero)
-                    Interop.CkmcTypes.BufferFree(ptr);
+                    CkmcTypes.BufferFree(ptr);
+            }
+        }
+
+        /// <summary>
+        /// Checks for alias existence
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <param name="alias">The name of a certificate to retrieve.</param>
+        /// <returns>Boolean indicating the exitsance of the alias.</returns>
+        /// <exception cref="ArgumentNullException">
+        /// The alias argument is null.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// Indicates failure to communicate with the Keystore. Check the errorcode for details
+        /// </exception>
+        static public bool AliasExists(string alias)
```

Comment:
It seem to need a password parameter
```suggestion
        static public bool AliasExists(string alias, string password = string.Empty)
```

Reply:
The function would return the same result regardless of the password correctness. It may confuse the caller. I'd rather keep it unchanged.

Reply:
What @kjackiewicz wrote is correct, we do not care whether we can decrypt the data. We only want to know whether it exists.